src/transaction/signTransaction.js

```diff
@@ -20,15 +20,43 @@ import serializeTransactionIntoCanonicalString from './serializeTransactionIntoC
  */
 export default function signTransaction(transaction, ...privateKeys) {
```

Comment:
What I'd like to see (maybe as tests):

What happens 
- if the wrong private keys get submitted to this method
- if too little or too much keys get submitted to this method

See also my comment on having a friend sign with you